input/blog/2017-12-29-cake-v0.24.0-released.md

```diff
@@ -0,0 +1,97 @@
+---
+title: Cake v0.24.0 released
+category: Release Notes
+author: devlead
+---
+
+Version 0.24.0 of Cake has been released.
+
+As a little holiday gift from the Cake team and contributors the last version for 2017 just shipped!
+This release shouldn't have any breaking changes, but it does have several improvements, bug fixes and a couple new features.
+
+A notable new feature is module bootstrapping built into the Cake script runner, which lets you with a `#module` preprocessor directive in your build script download and install modules from NuGet similar to what you've previously already been able to do with tools & addins.
+
+```cake
+#module nuget:?package=Cake.UrlLoadDirective.Module&version=1.0.2
+```
+
+Only caveat is that as modules can change and extend the internals of Cake, this bootstrapping needs to be done before Cake executes, this is solved by bootrapping being it's own step by invoking Cake with a `--bootstrap` argument before you execute the script normally.
```

Comment:
The only caveat...before Cake executes. This is solved by bootstrapping being it's own step triggered by invoking...